math/minuit/src/TFitter.cxx

```diff
@@ -137,8 +137,12 @@ void TFitter::GetConfidenceIntervals(Int_t n, Int_t ndim, const Double_t *x, Dou
    Double_t c=0;
 
    Double_t *matr = GetCovarianceMatrix();
-   if (!matr)
+   if (!matr){
+      delete [] grad;
```

Comment:
Hm, I wouldn't consider adding `grad = nullptr;` development :) but complete coverity fix reducing the possibility of introducing bugs...